website/src/components/UserChoice.tsx

```diff
@@ -0,0 +1,51 @@
+import { Card, CardBody, Flex, Heading } from "@chakra-ui/react";
```

Comment:
The  name `Choice` feels a little odd.  Is there a more intuitive name? Maybe something like `TaskOptions` as the component and then `TaskOption` for each individual thing that can be chosen.  It makes more sense to me at least to tie it to tasks rather than the user.